src/box/memtx_tx.h

```diff
@@ -536,12 +583,17 @@ memtx_tx_on_index_delete(struct index *index);
  * It's necessary because there is a chance that garbage collector hasn't
  * deleted all stories of that space and in that case some actions of
  * story's destructor are not applicable.
+ *
+ * NB: can trigger story garbage collection.
  */
 void
 memtx_tx_on_space_delete(struct space *space);
 
 /**
  * Create a snapshot cleaner.
+ *
```

Comment:
Since we removed garbage collection in this method we should remove the comment as well.

Reply:
Sorry, wasn't careful enough, fixed.